src/formatting.tsx

```diff
@@ -125,31 +126,27 @@ export function getNameOrDefault(obj?: unknown, { field = 'name', defaultValue =
   return defaultValue;
 }
 
-export function getOrDefault(value?: unknown) {
-  const isUndefined = value === undefined,
-    isNull = value === null,
-    isEmptyString = isString(value) && !hasStringContent(value);
-
-  if (isUndefined || isNull || isEmptyString) {
-    return EMPTY_FIELD;
+export function getOrDefault(value?: unknown): string {
+  if (hasStringContent(value)) {
+    return value.trim();
   }
 
-  if (isString(value)) {
-    return value.trim();
+  if (isNumber(value)) {
+    return String(value);
```

Comment:
This is the only functional change here, and it's reflected in the test suite.

Reply:
was this change necessary for the ant design upgrade?

Reply:
Nope. Just seemed like a good change. `getOrDefault` should probably return a string, right? If not, I can change it to `string | number`

Reply:
I would leave it as `string | number`, since I see this function as defaulting only if the value is empty. but don't feel strongly either way. doesn't look like it's being used for numbers currently.